usaspending_api/awards/migrations/0069_auto_20191226_1602.py

```diff
@@ -8,7 +8,7 @@
 class Migration(migrations.Migration):
```

Comment:
should this file be "moved" to 0002_auto_20191226_1602.py ?

Reply:
It could be.  I didn't seem the harm in leaving it untouched plus it was less work in that we wouldn't need to rename the migration in any databases where it may have aleady been applied.